src/mtype.cpp

```diff
@@ -50,6 +50,8 @@ mtype::mtype()
     phase = phase_id::SOLID;
     def_chance = 0;
     upgrades = false;
+    upgrades_multi = false;
+    upgrade_multi_range = cata::optional<int>();
```

Comment:
And why do you need both: an optional + a boolean? You could check for `upgrade_multi_range` containing a value instead of checking `upgrades_multi`.

Reply:
Cheers, removed `upgrades_multi`.